### PR TITLE
Move checkpoint files to S3

### PR DESCRIPTION
**Pull request by deirdre-k**
Adds `download_weights` to models:
* bp_resnet50_julios
* dbp_resnet50_julios
* resnet50_eMMCR_Vanilla

### CODE REVIEW

File: brainscore_vision/models/bp_resnet50_julios/model.py [hunk 1]
```diff
@@ -23,16 +23,15 @@ def get_model_list():
...
-    # save_directory=Path(__file__).parent)
-    # ckpt_path = os.path.join(os.path.dirname(__file__), 'ckpt.pth')
-    # state_dict = torch.load(ckpt_path, map_location=torch.device('cpu'))
-    # model.load_state_dict(state_dict)
-    
+    ckpt_name = 'bp_export.pt'
+    download_weights(
+        bucket='brainscore-vision',
+        folder_path='models/bp_resnet50_julios',
+        filename_version_sha=[(ckpt_name, 'PAH2BMGhcw9tWUH6RwTIADHodw.fyG4G', 'e504fc533aaec453f24e73a86e47ca6df3394c53')],
```
Comment (pellegreene) [inline brainscore_vision/models/bp_resnet50_julios/model.py:30]: Two questions:
Are these hashes static? I.e. will they ever be re-generated in S3? If so, is there a way to reference these without hard-coding?

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,8 @@
 *AWSCLIV2.pkg
 *.egg-info/
 venv
-.venv
\ No newline at end of file
+.venv
+.DS_Store
+
+# Model Weights
+brainscore_vision/models/**/*.pt
```
Comment (mschrimpf) [inline .gitignore:11]:
```suggestion
*.pt
```
I think we can remove `.pt` files altogether

File: brainscore_vision/models/bp_resnet50_julios/model.py (hunk 1 above)
Comment (mschrimpf) [reply brainscore_vision/models/bp_resnet50_julios/model.py:30]: as long as the file content does not change, the hash does not change

File: brainscore_vision/models/bp_resnet50_julios/model.py [hunk 3]
```diff
@@ -23,16 +23,15 @@ def get_model_list():
...
+        save_directory=Path(__file__).parent
+    )
+    ckpt_path = os.path.join(os.path.dirname(__file__), ckpt_name)
```
Comment (mschrimpf) [inline brainscore_vision/models/bp_resnet50_julios/model.py:33]:
```suggestion
    ckpt_path = Path(__file__).parent / ckpt_name
```
just a suggestion if you find this to be nicer

File: brainscore_vision/models/dbp_resnet50_julios/model.py [hunk 4]
```diff
@@ -23,16 +23,15 @@ def get_model_list():
...
-    # model.load_state_dict(state_dict)
-    
+    ckpt_name = 'dbp_export.pt'
+    download_weights(
+        bucket='brainscore-vision', 
+        folder_path='models/dbp_resnet50_julios',
+        filename_version_sha=[(ckpt_name, 'HpyXY.W.WtS8zD.wJ7BVfvSbQDX.1Md4', '6efa18bfe4b70221d7e88ac2d2279d08042f23da')],
+        save_directory=Path(__file__).parent
+    )
+    ckpt_path = os.path.join(os.path.dirname(__file__), ckpt_name)
```
Comment (mschrimpf) [inline brainscore_vision/models/dbp_resnet50_julios/model.py:33]:
```suggestion
    ckpt_path = Path(__file__).parent / ckpt_name
```

File: brainscore_vision/models/resnet50_eMMCR_Vanilla/model.py [hunk 5]
```diff
@@ -24,7 +24,14 @@ def get_model_list():
...
     model = torchvision.models.resnet50(pretrained=False)
-    ckpt_path = 'classifier.pt'
+    ckpt_name = 'classifier.pt'
+    download_weights(
+        bucket='brainscore-vision', 
+        folder_path='models/resnet50_eMMCR_Vanilla',
+        filename_version_sha=[(ckpt_name, 'KadF3fD38Y5yYBmZVLkOJzYW65k728nD', '073f002546a4e940179983222bc99c713febed88')],
+        save_directory=Path(__file__).parent
+    )
+    ckpt_path = os.path.join(os.path.dirname(__file__), ckpt_name)
```
Comment (mschrimpf) [inline brainscore_vision/models/resnet50_eMMCR_Vanilla/model.py:34]:
```suggestion
    ckpt_path = Path(__file__).parent / ckpt_name
```

**Review by mschrimpf — APPROVED** [review]
looks good as long as model tests pass!

let's also tag the original model submitter if we have their github

File: brainscore_vision/models/bp_resnet50_julios/model.py (hunk 1 above)
Comment (pellegreene) [reply brainscore_vision/models/bp_resnet50_julios/model.py:30]: I remember someone was working on model versioning for users who submit the same models multiple times. Would that precipitate a new hash?

**Review by mike-ferguson — APPROVED** [review]
lgtm pending martin's suggestions

**Comment by mike-ferguson** [comment]
Forced-merged per @deirdre-k (This PR is needed to unblock Jenkins tests, held up by unrelated test failures/timeouts)

**Pull request merged**